rules/cloud/aws/cloudtrail/aws_cloudtrail_console_susp_login.yml

```diff
@@ -0,0 +1,26 @@
+title: AWS Console Login Monitoring
+id: 313e72de-0c0d-4d65-8c95-87f4d546eceb
+status: test
+description: Detects successful AWS console logins and filters out legitimate countries and IP addresses. This alert helps identify potential unauthorized access attempts by monitoring successful console logins and excluding known legitimate locations.
+references:
+    - https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-event-reference-aws-console-sign-in-events.html
+author: Ivan Saakov
+date: 2024-09-21
+tags:
+    - attack.privilege-escalation
+    - attack.t1078.004
+logsource:
+    product: aws
+    service: cloudtrail
+detection:
+    selection:
+        eventName: ConsoleLogin
+        status: success
+    filter_country:
+        # Add legitimate countries src_country here
+    filter_ip:
+        # Add legitimate src_ip ranges here
```

Comment:
You can use a sigma rule like this. If you want to leave a placeholder for users to fill then please use a placeholder rule. See example https://github.com/SigmaHQ/sigma/tree/master/rules-placeholder and https://github.com/SigmaHQ/sigma-specification/blob/main/specification/sigma-rules-specification.md#placeholders